src/client/pages/AssessmentHome/FraHome/Links/Links.tsx

```diff
@@ -31,6 +31,10 @@ const Links: React.FC = () => {
   const countryFiles = assessmentFiles[countryIso] || []
   const globalFiles = assessmentFiles.globals
 
+  const isAdmin = Users.isAdministrator(user)
+
+  const isAllowedToEdit = Users.getRolesAllowedToEdit({ user, countryIso, cycle }).length > 0
```

Comment:
This checks that the user can edit other users ? How do we prevent that backend side ? Should we introduce a new check ?

Reply:
backend check is guaranteed by AuthMiddleware.requireEditAssessmentFile.

If countryIso is passed, only getRolesAllowedToEdit can add/delete a file.

If countryIso is not passed, only admins can add/delete a file.

I've verified it yesterday.